pkg/resolver/searchComplete.go

```diff
@@ -92,9 +111,17 @@ func (s *SearchCompleteResult) searchCompleteQuery(ctx context.Context) {
 		if err != nil {
 			klog.Errorf("Error building SearchComplete query: %s", err.Error())
 		}
+		f, err := os.Create("srchCompQuery.sql")
+		if err != nil {
+			fmt.Println("err creating file")
+		}
+		_, err = f.WriteString(sql)
+		if err != nil {
+			fmt.Println("err writing file")
+		}
```

Comment:
This looks like debug code. We could use a flag like `DEBUG_QUERY` if this can be helpful in the future.